test/unit/unit_tests_err.cpp

```diff
@@ -0,0 +1,73 @@
+
+#include <limits.h>
+#include "gtest/gtest.h"
+
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+#include "../../src/backends.h"
```

Comment:
Better to have a `-I` pointing to the `../..` and then go `#include src/backends.h"` or something.

Reply:
thank you for mentioning this Rafi. Adding the [`target_include_directories`](https://cmake.org/cmake/help/v3.12/command/target_include_directories.html#command:target_include_directories)